### PR TITLE
feat(Footer) : Footer page

### LINKED ISSUE #16: Important Pages and Section
- [ ] Privacy Policy page
- [ ] Refund page
- [ ] Footer

Okay, I am not sure where the old contents are, but will give it soon, make the footer for now.

### LINKED ISSUE #22: Loading page
Loading page is not properly displaying for Home page

### PR DESCRIPTION
**Pull request by Vyshnav001**

### CODE REVIEW

**Comment by anishshobithps** [comment]
> @Vyshnav001 is attempting to deploy a commit to the **gdscsjec-devs** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=gdscsjec-devs&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%2214fe5b850886c55aa8174ee261e57895782c53d9%22%7D%2C%22id%22%3A%22QmY4QgQ2eoAet38KuuBpDRWdjJfHFR7swoksdaJ9dSmbaM%22%2C%22org%22%3A%22tiarasjec%22%2C%22prId%22%3A20%2C%22repo%22%3A%22website%22%7D).

Not sure why is it asking me to authorize your PR everytime ?

**Comment by anishshobithps** [comment]
image

Make AR as red, and on Hover make Google DSC as Red only use the tiara_red, then add proper spacing between others.

To avoid confusion also rename one privacy policy as Refund Policy. Also there is Terms & Conditions.

**Comment by Vyshnav001** [comment]
For some reason Footer is not properly displaying in Event page

**Comment by anishshobithps** [comment]
> For some reason Footer is not properly displaying in Event page

We will see to it, once #15 is merged!

**Comment by Vyshnav001** [comment]
@anishshobithps  should i keep underline color also as red ?or white is fine ?

**Comment by anishshobithps** [comment]
> @anishshobithps should i keep underline color also as red ?or white is fine ?

Not sure if underline is going to look good ?

**Comment by rabilrbl** [comment]
> > @Vyshnav001 is attempting to deploy a commit to the **gdscsjec-devs** Team on [Vercel](https://vercel.com).
> > A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=gdscsjec-devs&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%2214fe5b850886c55aa8174ee261e57895782c53d9%22%7D%2C%22id%22%3A%22QmY4QgQ2eoAet38KuuBpDRWdjJfHFR7swoksdaJ9dSmbaM%22%2C%22org%22%3A%22tiarasjec%22%2C%22prId%22%3A20%2C%22repo%22%3A%22website%22%7D).
> 
> Not sure why is it asking me to authorize your PR everytime ?

He's using personal forked branch

File: src/components/widgets/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import SJECLogo from "@/assets/sjeclogo.png";
+import Image from "next/image";
+
+const Footer = () => {
+    return (
+        <footer className="text-gray-600 body-font w-full">
+            <div className="container w-full px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+                <a href="/" className="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
+                    <Image src={SJECLogo} alt="SJEC Logo" className="w-10 h-15" width={70}/>
+                    <span className=" font-tiara ml-3 text-xl text-white ">Ti</span><span className=" font-tiara text-xl text-tiara_red ">ar</span><span className=" font-tiara text-xl text-white ">a 2024</span>
```
Comment (anishshobithps) [inline src/components/widgets/Footer.tsx]: the font is quite tightly packed, use `leading-wider` iirc, that time text will be a bit wide and look better, Also I think we can remove SJEC logo at the footer ?  cc @rabilrbl 

Also, @rabilrbl will provide you with Privacy policy etc... make a separate route for it, as in like /privacy, /refund, /terms

You can use `prose` from `tailwindcss/typography` to format neatly.

**Review by anishshobithps — CHANGES_REQUESTED** [review]
Also, the contact number should be changed, and the email is `tiara@sjec.ac.in`,

File: src/components/widgets/Footer.tsx
Comment (rabilrbl) [inline src/components/widgets/Footer.tsx:1]: Can you please store all SVG icons under a folder in components?

File: src/components/widgets/Footer.tsx (hunk 1 above)
Comment (rabilrbl) [reply src/components/widgets/Footer.tsx]: @anishshobithps 
SJEC logo in footer not required

I only have T&C, come up with own privacy policy

**Comment by Vyshnav001** [comment]
1. phone number removed 
2. email id changed
3. removed SJEC logo from the footer
4. added link to privacy,refund,terms
5. made folder to store all the SVG icons

**Review by anishshobithps — APPROVED** [review]

**Pull request merged**